README.md

```diff
@@ -1,20 +1,34 @@
 # Code + Learn
 
-A series of workshop sprints for Node.js.
+Code + Learn is an in-person (and free) event series where developers receive mentoring from Node.js Core contributors to learn the step-by-step process of contributing to Node.js Core.
 
 ## Purpose
 
 Code & Learn events allow you to get started (or go further) with Node.js core
 contributions. Experienced contributors help guide you through your first (or
 second or third or fourth) commit to Node.js core. They also are available to
 provide impromptu guided tours through specific areas of Node.js core source
-code. 
+code.
 
-## Implementation
+## Getting started
 
-*[This is still actively under discussion.](https://github.com/nodejs/code-and-learn/issues/1)*
+- If you don't already have one, [create a GitHub account](https://github.com/join).
+- Open the [Node.js core repository web page](https://github.com/nodejs/node).
+- Use the "Fork" button to fork the repository:![Image](http://nodetodo.org/getting-started/zfork.png)
+- On the page _for your own fork_, use the "Clone or download" button and copy the URL for cloning to your clipboard:![Image](http://nodetodo.org/getting-started/zclone.png)
+- On the command line, in your home directory or a directory you have for projects, run:
 
-## Prior Art and Integration
+git clone &lt;paste that URL here!&gt;
+- When that is done:
 
-Obviously, NodeSchool is a widely distributed community that is teaching new Node developers. While there's a lot that we can learn from NodeSchool and we will be actively engaged with that community in order to expand the geographic presence of Code + Learn, the goals are slightly different. NodeSchool, by design, doesn't make much of an effort to onboard contributors into the Node.js core project. NodeSchool also relies on building material for use by groups which by definition can't address getting contributors involved in areas of the project which are not very well or widely understood.
+cd node
```

Comment:
Can you wrap these in backticks, or add full code fences, like this?

```js
    ```console
    cd node
    ```
```